app/views/pages/marketplace_setup/constants/set.liquid

```diff
@@ -0,0 +1,16 @@
+---
+method: post
+---
+{% liquid
+  include 'lib/must_be_superadmin'
```

Comment:
@damcik You have to first fetch current profile. Check other admin pages for example.